docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '3.2'
+
+services:
+  # recall to sudo sysctl -w vm.max_map_count=262144 on swarm nodes
+  elasticsearch:
+    image: martel/docker-swarm-elasticsearch:5.4.0
```

Comment:
Looks like here should be `agoryachev/docker-swarm-elasticsearch:5.4.0`

Reply:
hi, you are totally right :) of course having to test it, I replicated the image somewhere else, i will update right away.